suite-common/metadata-types/src/metadataTypes.ts

```diff
@@ -161,7 +165,7 @@ export type Labels = AccountLabels | WalletLabels;
 
 export type DeviceMetadata =
     | {
-          status: 'disabled' | 'cancelled'; // user rejects "Enable labeling" on device
+          status: 'disabled';
```

Comment:
why 'cancelled' status no more?

Reply:
[f228f6b](https://github.com/trezor/trezor-suite/pull/9130/commits/f228f6be38ab14be056817b6decbfc54721ae20c) I completely removed `device.metadata.status` and replaced it with `metadata.cancelledForDevice`. This also resolves the problem that caused metadata to become globally disabled when it got cancelled on a device